### ISSUE TITLE
Add glycan_torsion_angles code to examples repo

### ISSUE THREAD

**Issue by OriolAbril** [arviz-devs/arviz#1296]
If possible, add the code used to generate `glycan_torsion_angles` to https://github.com/arviz-devs/arviz_example_data. @agustinaarroyuelo

**Comment by aloctavodia** [comment]
@agustinaarroyuelo Did you use bomeba to generate the `glycan_torsion_angles` example, right? In that case I am a little bit worried about long term maintainability of the example. May be we could switch to a simpler to reproduce example.

**Comment by agustinaarroyuelo** [comment]
Yes, I did use Bomeba and the example is quite long. I have the same concerns as you

**Comment by aloctavodia** [comment]
ok then, let's think about a simpler example.
